src/main/java/org/apache/mesos/offer/OfferEvaluator.java

```diff
@@ -63,6 +63,20 @@ public OfferEvaluator(StateStore stateStore) {
         return Collections.emptyList();
     }
 
+    public List<OfferRecommendation> evaluate(OfferRequirement offerRequirement, Offer offer)
```

Comment:
I believe this function was removed because nothing called it.  Is it called today?